pkg/terraform/terraform.go

```diff
@@ -245,36 +245,58 @@ func RemoveDirectory(terraformDir string) error {
 // RunTerraform executes terraform apply function that creates the EC2 instances based on the .tf file provided
 // returns the AWS node-ID and node IP
 func RunTerraform(terraformDir string) (string, string, error) {
-	var instanceID string
-	var publicIP string
 	cmd := exec.Command(constants.Terraform, "init") //nolint:gosec
 	cmd.Dir = terraformDir
 	if err := cmd.Run(); err != nil {
 		return "", "", err
 	}
 	cmd = exec.Command(constants.Terraform, "apply", "-auto-approve") //nolint:gosec
 	cmd.Dir = terraformDir
-	utils.SetupRealtimeCLIOutput(cmd)
```

Comment:
can we make SetupRealtimeCLIOutput to setup two multiwriters, one for os.Stdout + stdoutBuffer and another
for os.Stderr + stderrBuffer, and return the two buffers?
Then we can assing one to stderrBuffer (not stdErr) and continue from there